apps/frontend/app/admin/problem/_components/ProblemsDownload.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+
+import { Button } from '@/components/shadcn/button'
+import { usePathname } from 'next/navigation'
+import { FiDownload } from 'react-icons/fi'
+
+type Mode = 'my' | 'shared'
+
+const API_BASE = process.env.NEXT_PUBLIC_BASEURL ?? ''
+
+export function ProblemsDownload() {
+  const pathname = usePathname()
+
+  const mode: Mode = pathname.startsWith('/admin/problem/shared')
+    ? 'shared'
+    : 'my'
+
+  const href = `${API_BASE}/problem/0/download?mode=${mode}`
+
+  const label =
+    mode === 'my' ? 'Download My Problems' : 'Download Shared Problems'
+
+  return (
+    <div className="mt-2 flex">
+      <Button asChild className="w-[250px]">
```

Comment:
으로 바꿔야 'Download Shared Problems'에서도 꽉차지 않아보여요!
```suggestion
      <Button asChild className="min-w-[250px]">
```